distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering/Comparator.enso

```diff
@@ -19,7 +19,7 @@ polyglot java import org.enso.base.ObjectComparator
      Otherwise can support a custom fallback comparator.
 new : Nothing | (Any -> Any -> Ordering) -> ObjectComparator
 new custom_comparator=Nothing =
-    comparator_to_java cmp x y = handle_incomparable_value (cmp x y . to_sign)
+    comparator_to_java cmp x y = Incomparable_Values_Error.handle_errors (cmp x y . to_sign)
```

Comment:
I'd like to point out that various `compare_to` methods throw different `Error`. Shouldn't all compare failures result into the same `Incomparable_Values_Error`?

Reply:
Yes that would be a fair step but out of scope for this PR.